.github/workflows/android-ci.yml

```
@@ -0,0 +1,47 @@
+concurrency:
+    group: build-${{ github.ref }}
+    cancel-in-progress: true
+
+jobs:
+    ci-build:
+        runs-on: ubuntu-latest
+
+        if: ${{ !contains(github.event.pull_request.labels.*.name, 'skip-ci') }}
+
+        steps:
+            -   name: Checkout
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)